### PR TITLE
Excluding build artifacts from copyright check

### PR DESCRIPTION
**Pull request by EricJohnson327**

### CODE REVIEW

File: dev/MRTCore/mrt/mrm/UnitTests/testPri.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (BenJKuhn) [inline dev/MRTCore/mrt/mrm/UnitTests/testPri.cpp:1]: Looks like you missed an "All rights reserved"?

Comment (BenJKuhn) [reply dev/MRTCore/mrt/mrm/UnitTests/testPri.cpp:1]: Actually, seeing at least three instances, might be worth a re-scrub.

**Review by BenJKuhn — APPROVED** [review]

File: tools/VerifyCopyrightHeaders.cmd [hunk 2]
```diff
@@ -1,14 +1,6 @@
...
 SETLOCAL
 
-IF /I "%1" == "-help" GOTO Help
-IF /I "%1" == "--help" GOTO Help
-
 powershell -ExecutionPolicy Unrestricted -NoLogo -NoProfile %~dpn0.ps1 %*
-GOTO TheEnd
-
-:Help
-powershell -ExecutionPolicy Unrestricted -NoLogo -NoProfile -c Get-Help %~dpn0.ps1 -full
```
Comment (DrusTheAxe) [inline tools/VerifyCopyrightHeaders.cmd:11]: How do I get usage help?

`VerifyCopyrightHeaders --help` ?

`VerifyCopyrightHeaders -?` ?

Please don't say `VerifyCopyrightHeaders -c Get-Help`

File: tools/VerifyCopyrightHeaders.ps1 [hunk 3]
```diff
@@ -3,12 +3,12 @@ Param(
...
 
 $copyrightHeaderText = (
-    "Copyright (c) Microsoft Corporation.",
+    "Copyright (c) Microsoft Corporation and Contributors.",
     "Licensed under the MIT License."
 )
 
 $include = ('*.cs', '*.cpp', '*.h', '*.idl', '*.xaml')
-$exclude = [RegEx]'\\dev\\Detours\\'
+$exclude = [RegEx]'\\dev\\Detours\\|\\BuildOutput\\|\\obj\\|\\localpackages\\|\\packages\\|\\specs\\|\\temp\\|\\Debug\\|\\Release\\'
```
Comment (DrusTheAxe) [inline tools/VerifyCopyrightHeaders.ps1:11]: Missing exclusions:

* docs
* x86, x64, arm64 (e.g. `C:\source\repos\windowsappsdk\dev\PushNotifications\PushNotificationsLongRunningTask.ProxyStub\x64\Debug\NotificationsLongRunningProcess_h.h`)
* temp -- probably should be excluded

Are .dirs traversed? If so thyen .config, .github and .user probably should be excluded

**Review by DrusTheAxe — DISMISSED** [review]

**Comment by DrusTheAxe** [comment]
When is this run?

Does it run in nightly and release branches in monitor-and-report(fail?) mode?

When is it run in -Fix mode? Manually because a problem was discovered or something regular/automated?

File: tools/VerifyCopyrightHeaders.ps1 (hunk 3 above)
Comment (EricJohnson327) [reply tools/VerifyCopyrightHeaders.ps1:11]: - docs doesn't have any of the included file types.
- x86/x64/arm64 are taken care of with the debug/release exclusions
- temp is already excluded

**Comment by EricJohnson327** [comment]
> When is this run?
> 
> Does it run in nightly and release branches in monitor-and-report(fail?) mode?
It runs as part of the TransportPackage-Foundation builds.  It just monitors for failures and blocks the build. 
> When is it run in -Fix mode? Manually because a problem was discovered or something regular/automated?
Fix mode must be run manually and changes checked in.

File: tools/VerifyCopyrightHeaders.cmd (hunk 2 above)
Comment (EricJohnson327) [reply tools/VerifyCopyrightHeaders.cmd:11]: There is no -help.  You run the script.  If it finds issues, it tells you want to do next.

File: dev/MRTCore/mrt/mrm/UnitTests/testPri.cpp (hunk 1 above)
Comment (EricJohnson327) [reply dev/MRTCore/mrt/mrm/UnitTests/testPri.cpp:1]: I had checked for two spaces before "all rights reserved".  Fixed.

**Pull request merged**